dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilderUtility.h

```diff
@@ -70,3 +73,55 @@ inline PCWSTR GetWinSoundEventString(AppNotificationBuilder::AppNotificationSoun
         return L"ms-winsoundevent:Notification.Default";
     }
 }
+
+inline std::wstring Encode(winrt::hstring const& value)
+{
+    std::wstring encodedValue{};
+    for (auto ch : value)
+    {
+        switch (ch) {
+        case '%':
```

Comment:
See comment above. Using a map would make this so much more simpler and you could avoid the switch case.

Reply:
You could have something like this:
if (map.contains(ch))
{
    encodedValue.append(map[ch];
}
else
{
 encodedValue.append(ch);
}

Reply:
Fixed